### PR TITLE
Added estimated duration to the next executions widget

### PR DESCRIPTION
**Pull request by kvanzuijlen**

### CODE REVIEW

**Comment by kvanzuijlen** [comment]
@ialbors?

File: src/main/resources/hudson/plugins/nextexecutions/NextExecutionsWidget/index.jelly [hunk 1]
```diff
@@ -26,9 +26,9 @@
       <tr>
         <td><a tooltip="${w.name}" href="${w.url}">${w.name}</a></td>
         <td tooltip="${w.timeToGo}">${w.date}</td>
+        <td tooltip="${w.
```
Comment (ialbors) [inline src/main/resources/hudson/plugins/nextexecutions/NextExecutionsWidget/index.jelly]: Something's missing here? :-)

Comment (kvanzuijlen) [reply src/main/resources/hudson/plugins/nextexecutions/NextExecutionsWidget/index.jelly]: You're right. I'll fix this when I get home. Don't know what I was smoking 😶.

**Comment by kvanzuijlen** [comment]
I think it could be handy to research how many jobs are scheduled to run simultaneously. Maybe we could make it configurable whether it should be shown or not?

**Pull request closed without merging**